modules/caddyhttp/encode/encode.go

```diff
@@ -232,18 +232,18 @@ func (enc *Encode) Match(rw *responseWriter) bool {
 	return enc.Matcher.Match(rw.statusCode, rw.Header())
 }
 
-// Flush implements http.Flusher. It delays the actual Flush of the underlying ResponseWriterWrapper
+// FlushError is an alternative Flush returning an error. It delays the actual Flush of the underlying ResponseWriterWrapper
 // until headers were written.
```

Comment:
Nitpick: keep comment length balanced 

```suggestion
// FlushError is an alternative Flush returning an error. It delays the actual Flush of the underlying
 // ResponseWriterWrapper until headers were written.
```